jumeg/connectivity/con_viz.py

```diff
@@ -547,16 +547,26 @@ def plot_grouped_connectivity_circle(yaml_fname, con, orig_labels,
     NOTE: yaml order fix helps preserves the order of entries in the yaml file.
     """
     import matplotlib.pyplot as plt
+
     # read the yaml file with grouping
     if op.isfile(yaml_fname):
         with open(yaml_fname, 'r') as f:
             label_groups = yaml.safe_load(f)
     else:
         print('%s - File not found.' % yaml_fname)
         sys.exit()
+    if yaml_color_fname is None:
```

Comment:
Coudl we use this as the alternative grouping file? desikan_aparc_cortex_based_grouping_ck.yaml

Reply:
I'm thinking if there is a way to avoid using the yaml files at all, any ideas?

Reply:
desikan_aparc_cortex_based_grouping_ck.yaml just provides a slightly different order so it would not really work.

If we want to avoid yaml files altogether we would have to provide an ordered dict for the grouping and a dict for the colours.

Reply:
Yeah, would be a good idea to give option to use ordered_dict directly, but it is best to make that in a separate PR.